vercel/static/sakura.css

```diff
@@ -0,0 +1,222 @@
+/* Sakura.css v1.4.1
```

Comment:
Might want to add a [`linguist-vendored`](https://github.com/github/linguist/blob/master/docs/overrides.md) to the `.gitattributes` for this fiel.

Reply:
Maybe we should exclude the vercel folder all together. I'm afraid of users being confused on whether the tool is actually `completely offline` - as stated in the README

Reply:
Good point. I'm not sure if there's an effective difference, but maybe the website should be `linguist-documentation`?

Reply:
`linguist-documentation` seems more appropriate.